modules/images/regenerate-existing-images/load.php

```diff
@@ -8,6 +8,18 @@
  * @package performance-lab
  */
 
+// Require helper functions and specific integrations.
+require_once __DIR__ . '/helper.php';
```

Comment:
We should also include the class file here. This is currently missing and would result in fatal errors if anyone called e.g. `perflab_create_background_job()`.